projects.json

```diff
@@ -4764,6 +4871,21 @@
             "branch": ["release/5.9"],
             "job": ["source-compat"],
             "platform": "Darwin"
+          },
+          {
+            "issue": "rdar://125516040",
```

Comment:
This was fixed on `main` and is passing now; here's the PR: https://github.com/apple/swift/pull/72706